packages/itwin/grouping-mapping-widget/src/widget/icons/DatabaseInfo.tsx

```diff
@@ -0,0 +1,32 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import React from "react";
+import "./DatabaseInfo.scss";
+
+const DatabaseInfoIcon = (): JSX.Element => {
+  return (
+    <>
```

Comment:
should talk with itwinui team, @FlyersPh9 , re adding this icon to our lib.

also are the fragments needed here?

Reply:
Yeah I think we should add this icon for sure. 
Fragments removed.